core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -245,8 +245,8 @@ object ConsumerGroupCommand extends Logging {
     private def printState(group: String, state: Option[GroupState]): Unit = {
       // this method is reachable only for the new consumer option (where the given state is always defined)
       if (shouldPrintMemberState(group, Some(state.get.state), Some(1))) {
-        print("\n%-25s %-20s %s".format("ASSIGNMENT-STRATEGY", "STATE", "#MEMBERS"))
-        print("\n%-25s %-20s %s".format(state.get.assignmentStrategy, state.get.state, state.get.numMembers))
+        print("\n%-20s %-25s %-20s %s".format("COORDINATOR", "ASSIGNMENT-STRATEGY", "STATE", "#MEMBERS"))
+        print("\n%-20s %-25s %-20s %s".format(state.get.coordinator.id, state.get.assignmentStrategy, state.get.state, state.get.numMembers))
```

Comment:
Thanks for adding this. Do you think we should also print host and port? For example, maybe something like `host:port (id)`

Reply:
Sure, do you think the output of `Node.toString()` works?

Reply:
I was thinking about that. The addition of `rack` is kind of annoying since it will usually be null and it seems not too useful in this context. Maybe we could do `{host}:{port} (id:{id})`?

Reply:
Yeah, if that's okay I'll use `COORDINATOR (ID)` as the column header so we don't need to repeat `id:` in each column cell. I'll probably create a local function to get the output `{host}:{port} ({id})` since now we need to make sure the content does not overflow the width, and extend the width dynamically if necessary.

Reply:
Sorry, ignore that. There is only one row. So we should be okay :)